.github/workflows/build.yml

```diff
@@ -6,12 +6,18 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest]
+        nim:
+          - stable
+          - devel --latest
```

Comment:
devel is supposed to be good enough of which the binary is built everyday. `devel --latest` might be slower since it builds from source.